src/stringUtils.js

```diff
@@ -50,6 +50,12 @@ const renderDate = date => {
   return notPresentString
 }
 
+const formatDateToLongHand = date => {
```

Comment:
looks more like a date utility rather than a string utility to me :grin:

Reply:
I agree! However, there's a bit of date parsing going on in there already, so just placed it with its friends. Should clean that file up at some point but maybe not for this ticket?